src/transformer_deploy/utils/args.py

```diff
@@ -45,6 +45,13 @@ def parse_args(commands: List[str] = None) -> argparse.Namespace:
             "For private models, use `True` to use local cached token, or a string of your HF API token"
         ),
     )
+    parser.add_argument(
+        "-l",
```

Comment:
I'm not sure it's a good idea to use short flags like `-l` for such specific command. Is it possible to only have `--load-external-data` ?

Reply:
https://docs.python.org/3/library/argparse.html#action

Reply:
@gaetansnl argparse.BooleanOptionalAction works only with python3.9 or higher and I think we did not impose this versions for transformer-deploy project.

Reply:
ok so we can implement the flags manually